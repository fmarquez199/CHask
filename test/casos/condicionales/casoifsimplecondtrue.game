Bl puedeConducir = F
In edad = 18
| edad >= 18:
  puedeConducir = T
