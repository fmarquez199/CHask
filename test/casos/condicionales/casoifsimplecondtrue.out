Token variable Bl en la fila: 1, columna: 1
Token variable puedeConducir en la fila: 1, columna: 4
Token = en la fila: 1, columna: 18
Token variable F en la fila: 1, columna: 20
Token variable In en la fila: 2, columna: 1
Token variable edad en la fila: 2, columna: 4
Token = en la fila: 2, columna: 9
Token entero 18 en la fila: 2, columna: 11
Token | en la fila: 3, columna: 1
Token variable edad en la fila: 3, columna: 3
Token >= en la fila: 3, columna: 8
Token entero 18 en la fila: 3, columna: 11
Token : en la fila: 3, columna: 13
Token variable puedeConducir en la fila: 4, columna: 3
Token = en la fila: 4, columna: 17
Token variable T en la fila: 4, columna: 19
